### ISSUE TITLE
Can’t create new note on mobile

### ISSUE THREAD

**Issue by curtismchale** [anpigon/obsidian-book-search-plugin#44]
When I try to create a new note with the Book Search action I get the following error.

```
TypeError: kr.join is not a function. (In 'kr.join(this.settings.folder,i)'; 'kr.join' is undefined)
```

Using the Book Search command to add data to a note works fine.

**Comment by anpigon** [comment]
@curtismchale Fixed in version 0.5.7. This issue is resolved by updating the plugin.

**Comment by curtismchale** [comment]
@anpigon confirmed. I checked for more updates a few days ago but didn’t get around to reporting till today. Sorry for wasting your time.
